prepare-image.sh

```diff
@@ -25,7 +25,8 @@ if [[ $INSTALL_TYPE == "source" ]]; then
     dnf upgrade -y
     # NOTE(dtantsur): pip is a requirement of python3 in CentOS
     dnf install -y python3-pip $BUILD_DEPS
-    pip3 install pip==21.3.1
+    pip3 install "pip"
```

Comment:
what is the reason to keep pip on this version ?

Reply:
@Rozzii no worries! :)
the reason to pin the version is that automatically getting the most recent version could bring in unexpected changes or bugs, that's also why I'd rather upgrade it in a separate change
this is how we usually do also in other projects to prevent "surprises"